test/helpers/kubectl.go

```diff
@@ -2857,6 +2857,21 @@ func (kub *Kubectl) GetCiliumPodOnNode(namespace string, node string) (string, e
 	return res.Output().String(), nil
 }
 
+// GetCiliumPodOnNode returns the name of the Hubble client pod that is running
+// on / in the specified node / namespace.
+func (kub *Kubectl) GetHubbleClientPodOnNode(namespace string, node string) (string, error) {
+	filter := fmt.Sprintf(
+		"-o jsonpath='{.items[?(@.spec.nodeName == \"%s\")].metadata.name}'", node)
+
+	res := kub.ExecShort(fmt.Sprintf(
+		"%s -n %s get pods -l k8s-app=hubble-cli %s", KubectlCmd, namespace, filter))
+	if !res.WasSuccessful() {
+		return "", fmt.Errorf("Cilium pod not found on node '%s'", node)
```

Comment:
I know it's existing code but would be great to add:

Add `: %s", node, res.OutputPrettyPrint())`